tests/layer_tests/tensorflow_tests/test_tf_Add.py

```diff
@@ -230,7 +231,7 @@ def test_add_placeholder_const_broadcast_3D(self, params, ie_device, precision,
         # ScaleShift
         dict(x_shape=[1, 3, 1, 1], y_shape=[3]),
         # ScaleShift
-        dict(x_shape=[1, 3, 100, 224], y_shape=[3]),
+        dict(x_shape=[1, 224, 100, 3], y_shape=[3]),
```

Comment:
why did you permute? I thought test cases are described in NCHW layout.

Reply:
@rkazants 
this test use broadcast, 
in case of using legacy, we perform transpose in the test body and inputs will have the shapes [1, 224, 100, 3] and [3] 
in case of new frontend, we don't use transpose and inputs will have the shapes [1, 3, 100, 224] and [3], we can't broadcast them --> FAIL

I suggest to process these cases separately, e.g:
```
        # we do not perform transpose in the test in case of new frontend
        if params['x_shape'] == [1, 3, 5, 1, 2] and params['y_shape'] == [5, 3, 2, 1] and use_new_frontend:
            params['x_shape'] = [1, 5, 3, 1, 2]
```